sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -87,6 +88,18 @@ static RT::PiProgram createSpirvProgram(const RT::PiContext Context,
   return Program;
 }
 
+static void getContextDevices(const RT::PiContext &Context,
+                              std::vector<RT::PiDevice> &Devices) {
+  size_t NumDevices = 0;
+  PI_CALL(piContextGetInfo)
+  (Context, PI_CONTEXT_INFO_NUM_DEVICES, sizeof(NumDevices), &NumDevices,
```

Comment:
Please apply clang-format. We should have some tweak in its config to properly format these strange `PI_CALL` stuff as regular call.

Reply:
This formatting is correct according to our config & the latest clang-format.